src/styles/blocks/catalog.scss

```
@@ -18,8 +18,17 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 
+    transform 0.3s ease, 
+    color 0.3s ease;
```

```suggestion
    transition: 
      transform 0.3s ease, 
      color 0.3s ease;
```
It's better to add indentation.